testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -0,0 +1,73 @@
+// see also TestServer_clientID
+/*
+TestServer_clientID_booted.run;
+*/
+
+// with running server
+TestServer_clientID_booted : UnitTest {
+
+	test_default {
+		var s = Server(this.class.name);
+		this.bootServer(s);
+		this.assertEquals(s.clientID, 0, "clientID should be 0 by default");
+		s.quit;
+		s.remove;
+	}
+
+	test_setWithSetter {
+		var options = ServerOptions.new.maxLogins_(4);
+		var s = Server(this.class.name, nil, options);
+		s.clientID = 3;
+
+		this.bootServer(s);
+		this.assertEquals(s.clientID, 3, "clientID should be settable by clientID_ before boot.");
+		s.quit;
+		s.remove;
+	}
+
+	test_setWithConstructor {
+		var options = ServerOptions.new.maxLogins_(4);
+		var s = Server(this.class.name, nil, options, 3);
+
+		this.bootServer(s);
+		this.assertEquals(s.clientID, 3, "clientID should be settable by Server constructor.");
+		s.quit;
+		s.remove;
+	}
+
+	// TODO: find a better home for this test
+	test_nodeIDAlloc_ServerTree {
+		var s = Server(this.class.name);
+		var synth1, synth2;
+		var func = { synth1 = Synth("default"); };
+
+		ServerTree.add(func, s);
+		this.bootServer(s);
+
+		synth2 = Synth("default", [\freq, 330]);
```

Comment:
let's try to avoid adding new noises to the test suite.

Reply:
IMO it's up to you to change it then; there's been no agreement about that and everyone else is fine with it